### DISCUSSION TITLE
2022.10.29 브리핑

### DISCUSSION THREAD

**Discussion by swjiae** [itmakesmesoft/Steady-Study#138] [Self-Assessment]
월간 목표

이은혁 : 토이 프로젝트 만들어보기
고영석 : 일주일에 알고리즘 문제 20개 이상 풀기, IM 통과하기
김지애 : 알고리즘 15개 이상 블로그 작성, sqld공부, cs공부
김지오 : IM 통과, 블로그 글 누적 60개, 다이어트(!), sqld 공부
나혜승 : IM 통과하기
이수연 : 생활코딩 자바 입문 수업 1/2 수강, 블로그 30개 이상 포스팅
이문삼 : 알고리즘, CS 각 15개 이상 블로그에 포스팅, 자바스크립트 강의 완강하기, 리액트 강의 시작하기
정혜수 : 블로그 포스팅+알고리즘, Django, 운동 꾸준히
김준영 : A형 통과하기

이번 주 목표 및 달성률(2022.10.23 ~ 2022.10.29)

이은혁 : velog 글 60개 , sqld 1회독
고영석 : 1일 1알고+블로그 작성, 진도 따라가기, sqld 조금씩 공부하기,
김지애 : sqld 1회독, 1일 1알고리즘, 진도 따라가기
김지오 : 블로그 글 60개, 진도 따라가기, 1일 1알고리즘, SQLD 진짜시작
나혜승 : SQLD 1회독 하기, 알고리즘 두문제 풀기
이수연 : sqld 1회독, 1일 1알고리즘
이문삼 :
정혜수 :
김준영 :

이번 주 목표를 이루기 위해 어떤 노력을 했는지 스스로 평가해보고, 점수를 매겨주세요.
ex)
이번 주 목표 : 기술 블로그에 배운 내용 정리하기

점수 : 80점
이유 : (생략)

**Comment by itmakesmesoft** [comment]
### 월간 목표
토이 프로젝트 만들어보기
점수 : 50점
이유 : 아직 제대로 구현이 안됨. 사실 구현 방법을 모른다가 정답임.
추후 vuejs를 이용해 구현해볼 수 있을지 않을까 기대..

### 주간 목표
velog 글 60개 , sqld 1회독
점수 : 70점
이유 : 블로그 글 60개 달성완료. but, sqld 아직 다 못풂...
내일까지는 1회독 하겠음..

**Comment by sooyeonlee127** [comment]
월간 목표
생활코딩 자바 입문 수업 1/2 수강, 블로그 30개 이상 포스팅
점수: 90점 자바 기초 절반은 수강했음. 블로그 30개 초과 포스팅 했음 퀄리티가 낮아서 10점 깎습니다..

주간 목표
sqld 1회독, 1일 1알고리즘

점수 : 70점
이유 : 1일 1알고리즘 성공, sqld 1회독 실패 .. 주말동안 1회독 끝내도록 노력하겠습니다..

**Comment by YoungSeok222222** [comment]
### **월간 목표**


일주일에 알고리즘 문제 20개 이상 풀기, IM 통과하기
점수: 50점
이유: IM은 통과했지만, 1일 1알고를 지키지 못했습니다. 다음 월간 목표는 지킬 수 있는 걸로 정하겠습니다.

### **이번주 목표:**
1일 1알고+블로그 작성, 진도 따라가기, sqld 조금씩 공부하기,
점수: 40점

이유: 먼저 1일 1알고 + 블로그 작성은 3알고 밖에 못했습니다. 진도는 따라가기는 했고, sqld는 포장지 뜯는데 성공했습니다. 더욱 노력하겠습니다 .감사합니다.

**Comment by swjiae** [comment]
월간목표 : 알고리즘 15개 이상 블로그 작성, sqld공부, cs공부
점수 : 60점
이유
sqld  아직 2과목 공부중..
cs공부 db 공부 하는 중!

이번 주 목표 : sqld 1회독, 1일 1알고리즘, 진도 따라가기

점수 : 60점
이유
sqld  아직 2과목 공부중..
1일 1알고 빠진날도 좀 있지만 최대한 노력중..
진도 이번주 수요일부터 놓쳤음..

**Comment by moonthree** [comment]
**월간 목표**
알고리즘, CS 각 15개 이상 블로그에 포스팅, 자바스크립트 강의 완강하기, 리액트 강의 시작하기
점수 : 40점
이유 : CS 15개 이상 블로그 포스팅 실패, 자바스크립트 강의 듣는중, 리액트 강의 듣는중

**주간 목표**
진도 따라가기, sqld 공부하기
점수 : 80점
이유 : 진도 잘 따라갔고, sqld는 오늘 열공할 예정입니다.

**Comment by HyeseungNA** [comment]
월간 목표: IM 통과하기
주간 목표 :SQLD 1회독 하기, 알고리즘 두 문제 풀기
점수:  60점 (im 통과했습니다. 알고리즘은 두문제 풀었고 SQLD는 아직 1회독 못했습니다...ㅠㅠ)

**Comment by Chung-HyeSu** [comment]
월간목표 : 블로그포스팅+알고리즘. Django. 운동꾸준히  -> 80점
이번주 목표 : 여러가지 행사로 완전 뻗어서 거의 못했다. 그래서 격려차 110점
주간목표 : sqld 공부.  수업진도 따라가기. 여러모로 습관을 다시 잡고, 시간을 밀도있게 쓰기.

다음 월간목표 : Web 공부 다지기. 포스팅(알고리즘).  습관.

**Comment by giokim12** [comment]
월간목표 : IM 통과, 블로그 글 누적 60개, 다이어트(!), sqld 공부
주간목표 : 블로그 글 60개, 진도 따라가기, 1일 1알고리즘, SQLD 진짜시작

라이브 수업은 다 들었지만 진도를 따라간다고 말하기는 어렵다
sqld도 시작 안했기 때문에 주간목표는 50점

월간목표도 im 통과랑 블로그 글은 썼지만 다이어트랑 sqld 공부는 안했기 때문에 50점
